src/JuliaFormatter.jl

```diff
@@ -37,24 +41,25 @@ include("styles/yas.jl")
 normalize_line_ending(s::AbstractString) = replace(s, "\r\n" => "\n")
 
 """
-    format_text(
-        text::AbstractString;
-        indent::Int = 4,
-        margin::Int = 92,
-        style::AbstractStyle = DefaultStyle(),
-        always_for_in::Bool = false,
-        whitespace_typedefs::Bool = false,
-        whitespace_ops_in_indices::Bool = false,
-        remove_extra_newlines::Bool = false,
-        import_to_using::Bool = false,
-        pipe_to_function_call::Bool = false,
-        short_to_long_function_def::Bool = false,
-        always_use_return::Bool = false,
-        whitespace_in_kwargs::Bool = true,
-    )::String
-
-Formats a Julia source passed in as a string, returning the formatted
-code as another string.
+```
+format_text(
+    text::AbstractString;
+    indent::Int = 4,
+    margin::Int = 92,
+    style::AbstractStyle = DefaultStyle(),
+    always_for_in::Bool = false,
+    whitespace_typedefs::Bool = false,
+    whitespace_ops_in_indices::Bool = false,
+    remove_extra_newlines::Bool = false,
+    import_to_using::Bool = false,
+    pipe_to_function_call::Bool = false,
+    short_to_long_function_def::Bool = false,
+    always_use_return::Bool = false,
+    whitespace_in_kwargs::Bool = true,
+)::String
+```
```

Comment:
When you look at the docstring via help in the REPL or docs generated from Documenter is the output identical to how this formatted previously?

Reply:
Also I don't think the code in these blocks is formatted, would that be possible?

Reply:
This PR only formats code as julia if it is marked as `@example`, `@repl`, `@eval`, `jldoctest`. I could change it to assume that any code without a language listed is julia?